lakey_service/account/models.py

```diff
@@ -67,7 +67,8 @@ def attach_account(self, email, oauth_token):
         self.account, _ = Account.objects.get_or_create(email=email)
         self.save()
 
-    def get_oauth2_email(self, oauth_token):
+    @staticmethod
+    def get_oauth2_email(oauth_token):
```

Comment:
you change here to `staticmethod` but I can't see changes to the places where `get_oauth2_email` is  called as a plain method, did you run tests and linter?

Reply:
I don't really understand the issue here. You can call staticmethods directly on class (i.e. even without instantiating an object), but also on the object of a class. As such, all existing implementation should continue to work with no issues. Or am I missing sth here... :thinking: 

flake8 linter was not run (due to issues discussed some time ago), all tests pass.